### PR TITLE
Make Metrics Example Flake8 Compliant

### PR DESCRIPTION
**Pull request by mharris-dstl**
Changes documentation in metrics example. I assume this was originally written in PyCharm where the default character limit is 120.

### CODE REVIEW

File: docs/examples/Metrics.py [hunk 1]
```diff
@@ -7,49 +7,55 @@
 """
 
 # %%
-# This example demonstrates some of the different metrics available in Stone Soup and how they can be used with the
-# :class:`~.MultiManager` to assess tracking performance. It also demonstrates how to use the :class:`~.MetricPlotter`
-# class to easily generate metric plots for visualisation.
+#
```
Comment (csherman-dstl) [inline docs/examples/Metrics.py:10]: Potentially remove blank line.

File: docs/examples/Metrics.py [hunk 2]
```diff
@@ -7,49 +7,55 @@
...
 from stonesoup.metricgenerator.basicmetrics import BasicMetrics
 
 basic_EKF = BasicMetrics(generator_name='basic_EKF', tracks_key='EKF_tracks', truths_key='truths')
 basic_PF = BasicMetrics(generator_name='basic_PF', tracks_key='PF_tracks', truths_key='truths')
 
 # %%
-# Next we create the Optimal SubPattern Assignment (OSPA) metric generator. This metric is calculated at each time step
-# to show how far the tracks are from the ground truth paths. It returns an overall multi-track to multi-ground-truth
-# missed distance for each time step.
 #
```
Comment (csherman-dstl) [inline docs/examples/Metrics.py:52]: Potentially remove blank line.

File: docs/examples/Metrics.py [hunk 3]
```diff
@@ -485,10 +507,13 @@
 graph.set_ax_title(['Extended Kalman Filter', 'Particle Filter'])  # set title for each axis
 
 # %%
-# For both trackers, we start with the highest uncertainty as tracks are initiated. Uncertainty decreases until just
-# past the middle of our timeframe and begins to increase again as we get further from the start time.
 #
```
Comment (csherman-dstl) [inline docs/examples/Metrics.py:510]: Potentially remove blank line

**Review by csherman-dstl — APPROVED** [review]
Just a couple places where a blank # at the start of a section could be removed.

**Review by jswright-dstl — APPROVED** [review]

**Pull request merged**